scripts/lint-yaml.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Use yamllint to lint and report errors for modified YAML files in repo
+# $ ./lint-yaml.sh
+
+set -e
+
+if [ -z "$(which yamllint)" ]; then
+    echo "yamllint not found" >&2
+fi
+
+RESULT=0
+
+INPUT=$1
+
+YAMLLINT_PARAMS="{extends: relaxed, rules: {line-length: disable, indentation: disable, colons: disable, empty-lines: disable, trailing-spaces: disable, new-line-at-end-of-file: disable }}"
```

Comment:
Can you pass a `.yamllint` file and enable disable rules in a separate config?

Looks like most of the rules are disabled here and you probably want it strict enough. I would also enable `truthy`.